js/analytics.md

```diff
@@ -100,6 +100,28 @@ Amplify.configure({
 User session data is automatically collected unless you disabled analytics. To see the results visit the [Amazon Pinpoint console](https://console.aws.amazon.com/pinpoint/home/).
 {: .callout .callout--info}
 
+#### Update your IAM Policy:
+
+AWS Pinpoint has updated their service and so you need to update your IAM policy in order to continue using the `record` API.
```

Comment:
Chance to:

Amazon Pinpoint service requires an IAM policy in order to use the record API: